### DISCUSSION TITLE
Seattle Fire 911 Calls

### DISCUSSION THREAD

**Discussion by NahomAgidew** [jakobzhao/geog495#54] [Final Project]
**Application:** https://nahomagidew.github.io/geog-495-final-project/

**GitHub:** https://github.com/NahomAgidew/geog-495-final-project

**Project Description:** This project presents a map of "911 calls" made to the Seattle Fire Department in the month of November 2021. The web app is able to filter the incidents by time and day of the call as well as specify what type of response was dispatched. Additionally, users can click on each dot to see a pop up of details such as address and time of the call.

![image](https://user-images.githubusercontent.com/11695305/145515375-09dd0ec9-6163-4af1-8f4f-ce38f01fb5a8.png)

**Comment by kevography** [comment]
I think this is a really well designed map! I think that for the most part the map is really readable. The only hiccup I think you might have is that the green icon for low acuity response sometimes blends in to the basemap, may be consider making it brighter similar to the other points?

**Comment by zeyany2** [comment]
Hi! This map is great, and I really like your idea of collecting all 911 calls. One suggestion I have is you can let user choose one type of calls instead of all types. All these colorful markers shown on map at the same time could make users hard to see what they are looking for.

**Comment by Stephenwzy** [comment]
Really a great application! It is interesting to visualize the pattern of 911 calls. The little dot markers make it easier to determine the distribution of different response types. One idea is that maybe you can make a filter for different types so that users can see each of types and make a comparison.

**Comment by Subarashiihibi** [comment]
I like your combination of basemap and the dots layer, which could make emphasis on your data in the map. I think something could be clarified is the legend of the map. I thought the numbers next to the legends were able to change relative to time, but it stayed unchanged so I feel if you could explain it more if it is total number of all cases at the location or all the same color are added up to the number.

**Comment by mhan799** [comment]
HI, I think this map is great at its base map design, which makes the patterns stand out. I think something that could be improved is that the color of Aid Response and Auto Fire Alarm as well as Medic Response and Transfer to AMR is a little alike and maybe change them to make the map more clear.

**Comment by yodapancake** [comment]
I like the map and really cool dataset you found. I like how the basemap is night themed. One thing that could be improved is adding the ability to put all the data on the map at once not just all the data of a given hour in the month/weedays/weekends. This would be interesting because it would allow the user to see where the most dangerous spots in Seattle are based on where clusters of types of calls are made very easily. It would also be great if you could filter the data on the map by type of call.

**Comment by Zhiyuan-Jiao** [comment]
Hello. I really like how you design your basemap to match up with all the venues. They make a strong contrast. One thing I think could be improved in this application is to make the incidence response type a filter function so that the user could select on with type they want to check out.

**Comment by FaisalTF** [comment]
This is definitely pushing the ceiling of my favorite projects as I'm browsing submissions! I think its very insightful with plenty to be analyzed and potentially further looked into, as well as think that the aesthetics are really nice. Trying to constructively think about how this could improve, I think playing around with the opacity of points could make it easier to digest some parts, especially as it gets somewhat congested by zooming out.

**Comment by RensenJiang** [comment]
Hi, it is an interesting map to help people visualize Seattle's 911 call. One thing to note is the color scheme you chose is hard to see for certain spots.
